### PR TITLE
Create QAM apache regression test

### PR DESCRIPTION
**Pull request by pdostal**
- Related ticket: [poo#52145](https://progress.opensuse.org/issues/52145)
- Verification run: [SLES15](http://pdostal-server.suse.cz/tests/3404)

### CODE REVIEW

**Comment by dzedro** [comment]
Extend what is already done

File: tests/console/apache.pm [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # Even before the installation there should be htdocs so we can create the index
+    assert_script_run 'echo "index" > /srv/www/htdocs/index.html';
+
+    # Ensure apache is installed and stopped
+    if (script_run('rpm -qa | grep apache2', 30) == 0) {
```
Comment (dzedro) [inline tests/console/apache.pm]: default timeout is 30, you don't need to specify it

Comment (dzedro) [reply tests/console/apache.pm]: same everywhere

File: tests/console/apache.pm [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    assert_script_run 'rm /var/run/httpd.pid';
+    systemctl 'start apache2';
+    assert_script_run 'ps aux | grep "\-f /etc/apache2/httpd.conf" | grep httpd-prefork';
+
+    # Test both instances
+    assert_script_run 'curl http://localhost:80/';
+    assert_script_run 'curl http://localhost:8080/';
+
+    # Stop both instances
+    assert_script_run 'killall -TERM httpd2-prefork';
```
Comment (dzedro) [inline tests/console/apache.pm]: systemctl will not stop it properly ?

Comment (dzedro) [reply tests/console/apache.pm]: there is no service for it ... ok

Comment (pdostal) [reply tests/console/apache.pm]: There are two instances running at the same time - one is started and then stopped from systemd and the other is started manually and then killed over here...

**Pull request merged**